core/modules/installer/installer.browser.inc

```diff
@@ -588,7 +588,7 @@ function installer_browser_add_remove_queue_dialog_link_submit($form, &$form_sta
 function installer_browser_check_zip_loaded() {
   $zip_loaded = array_key_exists('zip', archiver_get_info());
   if (!$zip_loaded) {
-    backdrop_set_message(t('Sorry, it seems that the PHP .zip extension is not loaded on your server. You will not be able to download any projects using Project Installer until this is fixed. Please contact your website administrator.'), 'warning', FALSE);
+    backdrop_set_message(t('Sorry, it seems that the PHP .zip extension is not loaded on your server. You will not be able to download any projects using Project Installer until this is fixed.'), 'warning', FALSE);
```

Comment:
Might as well remove the "Sorry" and "it seems" bits here. Also a slight order change with the words "PHP" and ".zip" so it is more precise:

```suggestion
    backdrop_set_message(t('The .zip PHP extension is not loaded on your server. You will not be able to download any projects using Project Installer until this is fixed.'), 'warning', FALSE);
```

Reply:
Nit pick - it's not .zip, it's just zip.  (no leading dot.)